### DISCUSSION TITLE
Extension Manager 0.2.0

### DISCUSSION THREAD

**Discussion by mjakeman** [mjakeman/extension-manager#60] [Announcements]
The first feature update to Extension Manager.

Extension Manager is a native tool for browsing, installing, and managing GNOME Shell Extensions. It is written with GTK 4 and libadwaita.

To install and run the flatpak below, make sure you have the [gnome-nightly](https://wiki.gnome.org/Apps/Nightly) repository installed.

**It is recommended that you install Extension Manager via [Flathub](https://flathub.org/apps/details/com.mattjakeman.ExtensionManager) for automatic updates.**

## What's Changed
 - Dark theme and support for overriding the system colour scheme
 - Shows screenshots in extension listings
 - Shell version compatibility check
 - Localised into seven languages
 - Sort search results
 - Global extension on/off toggle
 - Displays user and system extensions separately
 - Performance and stability

**Full Changelog**: [See Changelog](https://github.com/mjakeman/extension-manager/blob/9b174a8ce11123e0859fd3298aab9df6dbee96bd/NEWS)

<hr /><em>This discussion was created from the release <a href='https://github.com/mjakeman/extension-manager/releases/tag/v0.2.0'>Extension Manager 0.2.0</a>.</em>

**Comment by ovflowd** [comment]
Super cool work! 🎉 🙌
